index.html

```diff
@@ -50,8 +50,8 @@ <h1>RDF*</h1>
         </div> 	
 
         <p>
-          <a href="https://w3c.github.io/rdf-star/">RDF* Home</a>, part of the 
-          <a href="https://www.w3.org/community/rdf-dev/">RDF dev community group.</a>
+          <a href="https://w3c.github.io/rdf-star/">Home of RDF-Star (née RDF*)</a>, a part of the 
```

Comment:
I generally agree that a latinized version of "RDF*" is required in all related documents, including this page. However, I am not sure that "RDF Star" becoming the primary name would make consensus. Something like

>  Home of RDF* (pronounced "RDF star")

would be less contreversial at the moment, IMHO

Reply:
That's an excellent compromise.

Reply:
fine...

Reply:
> That's an excellent compromise.

I am not convinced that this is a "compromise" (let alone an "excellent" one).

In any case it does not resolve #20

Reply:
+1 @akuckartz.  That said, as this was just meant to improve the homepage, I'm OK with it not resolving #20.